comp/snmptraps/forwarder/component.go

```diff
@@ -0,0 +1,16 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+// Package forwarder defines a component that receives trap data from the
+// listener component, formats it properly, and sends it to the backend.
+package forwarder
+
+// team: network-device-monitoring
+
+// Component is the component type.
+type Component interface {
+	Start()
```

Comment:
Can you use [Lifecycle hooks](https://uber-go.github.io/fx/lifecycle.html) instead and not provide this method in the interface?

Reply:
Done.